src/components/SubTask/SubTaskFilter/SubTaskFilterContainer.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { ButtonGroup } from 'react-bootstrap';
+import { subtaskFilterTypes } from '../subTaskFilterTypes';
+import SubTaskFilterButton from './SubTaskFilterButton';
+
+class SubTaskFilterContainer extends Component {
+  render() {
+    return (
+      <>
+        <span>Show: </span>
+        <ButtonGroup>
+          <SubTaskFilterButton type={subtaskFilterTypes.SHOW_ALL} />
```

Comment:
you can have subtaskFilterTypes in array and map all types here. Would be more usable in case you'll have additional types in future